packages/fiber/src/core/renderer.ts

```diff
@@ -297,6 +297,9 @@ function createRenderer<TCanvas>(roots: Map<TCanvas, Root>) {
       if (!target)
         throw `${name} is not part of the THREE namespace! Did you forget to extend? See: https://github.com/pmndrs/react-three-fiber/blob/master/markdown/api.md#using-3rd-party-objects-declaratively`
 
+      // Throw if an object or literal was passed for args
+      if (args && !Array.isArray(args)) throw 'args must be an array!'
```

Comment:
Maybe `the args prop must be an array!`

Just thinking, _stupid proof_

Reply:
That's a bit better, yes. I also shouldn't check whether `args` is falsy.